contrib/mesos/pkg/scheduler/components/framework/framework.go

```diff
@@ -128,14 +123,6 @@ func New(config Config) Framework {
 					return false
 				}
 
-				// the executor IDs must not identify a kubelet-executor with a group that doesn't match ours
```

Comment:
This is intentionally post-poned. We did not have a proper, full story for different command line flags before either. We just got stale slave with unusable executor. We have to think how to really solve this problem.